### ISSUE TITLE
Cannot start service on newer RPi platforms: update to pinctrl instead of using /sys/class/gpio directory

### ISSUE THREAD

**Issue by jacoblapenna** [skhg/juice4halt#4]
See [pull request](https://github.com/skhg/juice4halt/pull/3). You are using `/sys/class/gpio` to interact with GPIO25 in bash. `/sys/class/gpio` has been deprecated since 2015 and targeted for removal in 2020 (see [here](https://www.thegoodpenguin.co.uk/blog/stop-using-sys-class-gpio-its-deprecated/)).  `pinctrl` (see [here](https://github.com/raspberrypi/utils/blob/master/pinctrl/README.md)) is the official Raspberry Pi util for command line manipulation and reading of GPIO. New Raspberry Pi OS versions will not support your current scripts (e.g., the Raspberry Pi 5).

I have submitted a tested PR to update to `pinctrl`. Please review and merge or provide comments for changes.

NOTE: I have forked and updated my own repo for my use. But because [J4H officially points to this repo](https://juice4halt.com/download) from their webpage, you should merge for others to use.

**Comment by jacoblapenna** [comment]
See fork using `pinctrl` [here](https://github.com/jacoblapenna/juice4halt) in case this repo is no longer maintained.

**Comment by damdo** [comment]
Hey @jacoblapenna so you are using the juice4halt HAT on a Raspberry Pi 5? Does it work?
Do you think it would work also on a a Raspberry Pi 4?
Thanks!

**Comment by jacoblapenna** [comment]
@damdo yes, it works on an RPi 4. The caps are overkill. They'll last more than a minute. Per this original issue, you may have to get creative with the detect-and-shut-down script with newer RPi OS variants. If this repo doesn't work, try my fork. If that doesn't work, play around with the script.

**Comment by damdo** [comment]
@jacoblapenna thanks a lot! I appreciate it :) Will take a look!
